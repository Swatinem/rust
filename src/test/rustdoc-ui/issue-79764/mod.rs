// check-pass
// compile-flags:--test
// normalize-stdout-test: "src/test/rustdoc-ui" -> "$$DIR"
// normalize-stdout-test "finished in \d+\.\d+s" -> "finished in $$TIME"

// The literal `mod` in the doctest comments makes rustdoc emit a bogus
// `extern crate mod;` line which would break the doctests.
#![doc(test(no_crate_inject))]

/// ```
/// // ^ mod.rs line 10
/// assert_eq!(1, 1);
/// ```
pub mod a;
