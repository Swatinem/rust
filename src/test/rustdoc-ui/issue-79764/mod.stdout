
running 4 tests
test $DIR/issue-79764/mod.rs - a (line 14) ... ok
test $DIR/issue-79764/mod.rs - a (line 10) ... ok
test $DIR/issue-79764/a/mod.rs - a::another_mod (line 6) ... ok
test $DIR/issue-79764/a/mod.rs - a::another_mod (line 10) ... ok

test result: ok. 4 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in $TIME

